src/lib/edge-functions/registry.cjs

```diff
@@ -396,12 +406,15 @@ class EdgeFunctionsRegistry {
   }
 
   async scan(directories) {
+    logH2({ message: 'Loading netlify edge functions' })
     const functions = await this.bundler.find(directories)
 
     functions.forEach((func) => {
       EdgeFunctionsRegistry.logAddedFunction(func)
     })
 
+    functions.push(log())
```

Comment:
This seems wrong, pushing a log line or undefined (?) to the functions array.

Reply:
I specifically wrote at the top of the PR for no one to look at this yet :)

This is the most drafty PR I have ever opened and I haven't even got my bearings on the repo yet. It's all a draft and I will be addressing things like this.

Thanks for your comment, but please no more until the PR is out of draft.

Reply:
I know I was just looking at what could be the reason for the timeouts, and only found this. 
*disappears in smoke*

Reply:
Oh interesting! Oh yeah this a horrendous line of code isn't it :D Thanks for spotting it!